### ISSUE TITLE
Non-countable monsters not factored into drpg_monster_limit

### ISSUE THREAD

**Issue by KuromeKT** [Sumwunn/DoomRPG#78]
Non-countable monsters, such as the Lost Soul family of monsters, are not factored into the drpg_monster_limit variable. For maps that have a few thousand non-countable monsters, drpg_monster_limit does nothing to improve GZDoom's performance in practice. Since they're covered by DRPG's scripts to possess levels, auras and attributes, they should also be taken into account when drpg_monster_limit is removing monsters from the map until they're no longer over the threshold.

**Comment by Sumwunn** [comment]
Hmm, damn. That's a side effect of me reworking Lost Souls in relation to the Pain Elementals.

I think recall being be able to apply the killcount flag after map spawning is done. Have to play around with this again haha.

**Comment by Sumwunn** [comment]
Ok so I think I've finally solved this. I more-or-less reworked my previous Lost Soul rework.

Seems to be working. From my testing so far map-based Lost Souls count towards total monsters and award XP, whereas any from say a Pain Elemental, won't count towards anything. This is the intended behaviour.

Changes are pushed to experimental, and when I've tested everything a bit more thoroughly I'll merge to master.

Oh boy, these Lost Soul's have been a pain haven't they haha.
